js/search-algorithm.js

```diff
@@ -136,7 +136,7 @@ define([
 
 		onDataReady: function() {
 			var config = Adapt.course.get("_search");
-			if (!config || !config._isEnabled ) return;
+			if (!config || _.has(config, '_isEnabled') && config._isEnabled === false) return;
```

Comment:
this is the only necessary line as far as i can see.

could you change it to:
```
if (!config || config._isEnabled === false) return;
```
as if config._isEnabled doesn't exist it'll evaluate to ``undefined`` which is not ``=== false``

and remove the other checks?